libs/fluxc-tests/src/test/java/org/wordpress/android/fluxc/theme/ThemeStoreUnitTest.java

```diff
@@ -44,7 +44,7 @@ public void setUp() {
     public void testActiveTheme() throws SiteSqlUtils.DuplicateSiteException {
         final SiteModel site = SiteUtils.generateWPComSite();
         TestSiteSqlUtils.INSTANCE.getSiteSqlUtils().insertOrUpdateSite(site);
-        assertNull(mThemeStore.getActiveThemeForSite(site));
+        assertEquals(0, ThemeSqlUtils.getActiveThemeForSite(site).size());
```

Comment:
Super minor np, a potentially a matter of preference but I think this style is more readable?

```suggestion
        assertTrue(ThemeSqlUtils.getActiveThemeForSite(site).isEmpty());
```

No need to change :)

Reply:
It is, thanks! Actually, I like assertj assertions the most, so I went with 

```java 
assertThat(ThemeSqlUtils.getActiveThemeForSite(site)).isEmpty();
```

in 66cad907bd4d0b2fe35eb4088c710ddd4b3f8560 🙂